docs/json_rpc_api.md

```diff
@@ -214,10 +214,10 @@ the method name for further details such as parameter and return information.
 |   |   |
 |---|---|
 |Method|createrawtransaction|
-|Parameters|1. `transaction inputs`:  `(JSON array, required)` json array of json objects.<br />`hash`: `(string, required)` the hash of the input.</br> `vout`: (numeric, required) the specific output of the input transaction to redeem transaction.<br /><br />`[{"txid": "hash", "vout": n}, ...]`<br /><br />2. `addresses and amounts`: `(JSON object, required)` - json object with addresses as keys and amounts as values.</br>`address`:   `(numeric, required)` the address to send to as the key and the amount in DCR as the value.<br /><br />`{"address": n.nnn, ...}`|
+|Parameters|1. `transaction inputs`:  `(JSON array, required)` json array of json objects.</br> `amount`: `(numeric)` the previous output amount.<br />`hash`: `(string, required)` the hash of the input.</br> `vout`: (numeric, required) the specific output of the input transaction to redeem transaction.<br /><br />`[{"amount":n, "txid": "hash", "vout": n}, ...]`<br /><br />2. `addresses and amounts`: `(JSON object, required)` - json object with addresses as keys and amounts as values.</br>`address`:   `(numeric, required)` the address to send to as the key and the amount in DCR as the value.<br /><br />`{"address": n.nnn, ...}`|
```

Comment:
Missing a space before the `n` in `[{"amount":n, "txid": "hash", "vout": n}, ...]`.

Reply:
Now that there are two amounts, it should probably specifically call out that the latter amounts are outputs amounts.

Reply:
Finally, I noticed that the address description is inaccurate.  Notice it says:

`address: (numeric, required) the address to send to as the key and the amount in DCR as the value.`

Which indicates it should literally be the key `address`.  It's not, rather it's actually the address to send the specified amount to.